providers/base/units/touchpad/test-plan.pxu

```diff
@@ -37,14 +37,13 @@ unit: test plan
 _name: Touchpad tests
 _description: Touchpad tests
 include:
-    touchpad/basic-after-suspend                   certification-status=blocker
-    touchpad/detected-as-mouse-after-suspend       certification-status=blocker
-    touchpad/palm-rejection-after-suspend          certification-status=non-blocker
-    touchpad/continuous-move-after-suspend         certification-status=blocker
-    touchpad/singletouch-selection-after-suspend   certification-status=blocker
-    touchpad/drag-and-drop-after-suspend           certification-status=blocker
-    touchpad/multitouch-rightclick-after-suspend   certification-status=blocker
-    touchpad/multitouch-after-suspend              certification-status=blocker
+    after-suspend-touchpad/basic                                 certification-status=blocker
+    after-suspend-touchpad/palm-rejection                        certification-status=non-blocker
+    after-suspend-touchpad/continuous-move                       certification-status=blocker
+    after-suspend-touchpad/singletouch-selection                 certification-status=blocker
+    after-suspend-touchpad/drag-and-drop                         certification-status=blocker
```

Comment:
These ones do not show up in the [output you provided](https://pastebin.canonical.com/p/6qpsY7ZhF8/). Maybe you forgot to add the `also-after-suspend` flag to their "before suspend" jobs.

Reply:
thanks for pointing that out, I did forget to add the flag. I've added it and it will show up now.